openapi/openapi.yaml

```diff
@@ -70,9 +70,15 @@ paths:
       parameters: 
         - $ref: "#/components/parameters/limitParam"
         - $ref: "#/components/parameters/offsetParam"
+        - $ref: "#/components/parameters/geometryParam"
```

Comment:
I wouldn't hold up the PR over this, and it's sort of out of scope anyway but I wanted to flag we might want to reconsider this `somethingParam` naming convention we've adopted. It's nitpicky but it just feels overly verbose to tack `Param` onto everything when the refs are already namespaced by `#/components/parameters`, unless there is some other name collision problem I'm missing.